usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -157,6 +157,29 @@ def signal_handler(signal, frame):
     def get_broker_submission(self):
         self.db_cursor.execute(
             f"""
+                with publish_certify_history as (
+                    select
+                        distinct_pairings.submission_id,
+                        json_agg(
+                            json_build_object(
```

Comment:
I think `jsonb` is preferable.

`json_agg` -> `jsonb_agg`
`json_build_object` -> `jsonb_build_object`

According to the docs, the postgres field `JSONField` is actually a jsonb datatype: https://docs.djangoproject.com/en/2.2/ref/contrib/postgres/fields/#django.contrib.postgres.fields.JSONField

I assume the datatype conversion to USAspending table storage will be seamless but it is probably better to be consistent.